### PR TITLE
Fix a typo in experiment.rst

### PR DESCRIPTION
**Pull request by bweber-rebellion**
Thanks for the project! I found this small typo as I was going through the docs.

### CODE REVIEW

File: docs/source/experiment.rst [hunk 1]
```diff
@@ -6,14 +6,14 @@ Experiment
...
-An experiment is a low level abstraction that takes the cartesian product of possible inputs to
+An experiment is a low level abstraction that takes the Cartesian product of possible inputs to
 an LLM API. For example, the ``OpenAIChatExperiment`` accepts lists of inputs for each parameter
 of the OpenAI Chat Completion API. Then, it constructs and asynchronously executes requests
 using those potential inputs. An example of using experiment is `here <https://github.com/hegelai/prompttools/blob/main/examples/notebooks/BasicExperiment.ipynb>`_.
 
 There are two ways to initialize an experiment:
 
-  1. Wrap your parameters in ``list``s and pass them into the ``__init__`` method. See each class's
+  1. Wrap your parameters in ``list`` s and pass them into the ``__init__`` method. See each class's
```
Comment (steventkrawczyk) [inline docs/source/experiment.rst:16]: Why are you adding a space here? The intention is to use "`list`s" as a plural of `list`.

Comment (bweber-rebellion) [reply docs/source/experiment.rst:16]: Sorry, I should have put that picture in the OP! 😄

**Comment by steventkrawczyk** [comment]
Hey @bweber-rebellion thanks for checking out our repo and making a PR! I had one comment about a space you added, but the capitalization of "Cartesian" looks good. Let me know once that's resolved and we can merge.

File: docs/source/experiment.rst (hunk 1 above)
Comment (steventkrawczyk) [reply docs/source/experiment.rst:16]: Ah, thanks for catching that! How does it render with the space?

Comment (bweber-rebellion) [reply docs/source/experiment.rst:16]: Building it locally, I get this:
image
It looks like there's still a space there...

Comment (bweber-rebellion) [reply docs/source/experiment.rst]: Looks like escaping that space fixes the problem: 
image

Comment (steventkrawczyk) [reply docs/source/experiment.rst]: Awesome, thanks for doing this!

**Pull request merged**